src/nurse/migrations/0001_initial.py

```diff
@@ -50,7 +50,6 @@ class Migration(migrations.Migration):
                 ("prescribing_doctor", models.CharField(max_length=300)),
                 ("start_date", models.DateField()),
                 ("end_date", models.DateField()),
-                ("at_renew", models.BooleanField()),
```

Comment:
That's not how we update migrations
https://docs.djangoproject.com/en/4.2/topics/migrations/